### ISSUE TITLE
Compensating driven motion

### ISSUE THREAD

**Issue by awegsche** [adamkoval/SKEKB_TbT_analysis#3]
**Implementation**

_Beta-Beat.src_

The implementation of the compensation of driven motion in the Beta-Beat.src repository checks for the files `twiss_ac.dat` or `twiss_adt.dat` in the model file. If one _and only one_ of them is present, the corresponding compensation is prepared (pre-calculation of parameters of this compensation).

_omc3_

In the new repository **omc3** the implementation is a bit different (with the goal to make it more intuitive and self-consistent):
Giving the command-line argument `--compensation` with one of the following options: 
`"model", "none", "equation"`. Unless `"none"` is given the type of excitation is determined as before. The options specify the following behaviour:

- `"model"`: the compensation is a simple rescaling of the optics functions using the difference between the models
- `"equation"`: a more sophisticated equation is used to calculate the compensated phase advances

The existence of a `twiss_elements_ac.dat` file is for sure better for the accuracy of the final optics results but I was told that the difference is negligible.

**The model creator**

Ideally, a model creator should be implemented for an automatic creation of the different model files `twiss*.dat`. But this is better to be done in the new omc3 repository.

**Summary**

To sum up and clear things up a bit, the model folder should include the following files:

```
model1
 +-- twiss.dat
 +-- twiss_elements.dat
 +-- error_deffs.txt

model2
 +-- twiss.dat
 +-- twiss_elements.dat
 +-- twiss_ac.dat
 +-- twiss_elements_ac.dat
 +-- error_deffs.txt
```

where model1 is one without excitation and model2 has an ac-dipole-like excitation.

_Note_
Please note the typo in `error_deffs.txt`. It is intentional (and should be changed in the new repository) but for compatibility with external codes we have to keep it in Beta-Beat.src.
